lib/libyui_client/widgets/progressbar.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+module LibyuiClient
+  module Widgets
+    # Class representing a Progressbar in UI. It can be YProgressBar.
+    class Progressbar < Widgets::Base
+      # Returns the current value of progressbar.
+      # Gets value from 'value' parameter in JSON representation of YProgressBar.
+      # @return [Integer] current value of progressbar.
+      # @example Get progressbar value, with id 'initProg'
+      #   {
+      #     "class" : "YProgressBar",
+      #     "hstretch" : true,
+      #     "id" : "initProg",
+      #     "label" : "Disk",
+      #     "max_value" : 1000,
+      #     "value" : 666
+      #   }
+      #   app.progressbar(id: 'initProg').current_value
+      #	    # 663
+      def current_value
```

Comment:
just a suggestion, you can leave it as it is, but I think that it is less typing just call it 'value' from outside and still is easy to understand what it does.

Reply:
Done